kubedriver/helmclient/client.py

```diff
@@ -21,13 +22,21 @@
 HOOKS_PREFIX = 'HOOKS:'
 MANIFEST_PREFIX = 'MANIFEST:'
 
+# Helm 3
+NAME_PREFIX = 'NAME:'
+LAST_DEPLOYED_PREFIX = 'LAST DEPLOYED:' # Replaces RELEASED_PREFIX
+NAMESPACE_PREFIX = 'NAMESPACE:'
+STATUS_PREFIX = 'STATUS:'
+NOTES_PREFIX = 'NOTES:'
+
 class HelmClient:
     
     def __init__(self, kube_config, helm_version, tls=None):
         self.tmp_dir = tempfile.mkdtemp()
         self.tls = tls if tls is not None else HelmTls(enabled=False)
         self.__configure_helm(kube_config)
         self.helm = f'helm{helm_version}'
+        self.helm_version = helm_version
```

Comment:
just in case, wrap this in str() to ensure we have a string later on. 
```
self.helm_version = str(helm_version)
```

Just a safety measure in case one day deployment locations support non-string properties - as the helm version will appear like a number to most parsers but we want a string.